node/config/def.go

```diff
@@ -99,13 +99,13 @@ func DefaultFullNode() *FullNode {
 			},
 		},
 		Cluster: *DefaultUserRaftConfig(),
-		ActorEvent: ActorEventConfig{
-			EnableRealTimeFilterAPI: false,
-			EnableHistoricFilterAPI: false,
-			FilterTTL:               Duration(time.Hour * 24),
-			MaxFilters:              100,
-			MaxFilterResults:        10000,
-			MaxFilterHeightRange:    2880, // conservative limit of one day
+		Events: EventsConfig{
```

Comment:
hrm @raulk  -  I thought we agreed to move this section under `FEVM` ? then the rename make sense cuz its FEVM.Events?

Reply:
Yes, I believe this makes sense for now. While the Events subsystem is technically independent of FEVM, this is a user-facing configuration file and I think it leads to better product usability for node operators if all the new options they care about are nested within the Fevm group. We might need to rejigger this when builtin-actors are emitting events or with Wasm actors, but we can cross that bridge when we get to it, because I expect changes in the event subsystem anyway.